quickstarts/consolequickstart-install-cryostat.yaml

```diff
@@ -0,0 +1,146 @@
+  introduction: >-
+    Cryostat is a container-native Java application based on JDK Flight Recorder (JFR). You can use it to monitor the performance of the Java Virtual Machine (JVM) for containerized workloads that run on an OpenShift cluster.
+
+    1. You can deploy Cryostat in a container in an OpenShift project that hosts your containerized Java applications. 
+
+    1. You can create JVM targets that correspond to the JVM instances that you use to run your containerized workload. 
+
+    1. You can connect Cryostat to the JVM targets to record and analyze data about heap and non-heap memory usage, thread count, garbage collection, and other performance metrics for each JVM target.
+
+    To start using Cryostat, install the cert-manager Operator for Red Hat OpenShift and Red Hat build of Cryostat Operators on your cluster.
```

Comment:
This line isn't shown as separate paragraph:

![image](https://github.com/openshift/console-operator/assets/139310/0846d624-66fb-4f49-8bbe-f7f2eb39ab75)

One extra line above this sentence fix this:

```suggestion

        To start using Cryostat, install the cert-manager Operator for Red Hat OpenShift and Red Hat build of Cryostat Operators on your cluster.
```